src/components/CreateMenu.js

```diff
@@ -21,7 +21,7 @@ const propTypes = {
         PropTypes.shape({
             icon: PropTypes.func.isRequired,
             text: PropTypes.string.isRequired,
-            onSelected: PropTypes.func.isRequired,
+            onSelected: PropTypes.func,
```

Comment:
It's not clear to me why this change is necessary, would you mind explaining?

Reply:
Initially I thought I could use something like this: 

```jsx
menuItems={[
    {
        icon: Paperclip,
        text: 'Upload Photo',
    },
]}
```

I'll revert it back to required though